src/dotnet/App.Maui/MauiDiagnostics.cs

```diff
@@ -62,7 +60,7 @@ public static IServiceCollection AddMauiDiagnostics(this IServiceCollection serv
     private static LoggerConfiguration CreateSerilogLoggerConfiguration()
     {
         var configuration = new LoggerConfiguration()
-            .MinimumLevel.Is(LogEventLevel.Information)
+            .MinimumLevel.Is(LogEventLevel.Debug)
```

Comment:
I think better to return back min log level to Info

Reply:
Yep.

Reply:
Guys, we manage min level at MSFT logger config 
![image](https://github.com/Actual-Chat/actual-chat/assets/5853608/e127bbcb-6909-4339-878d-58712532be71)

Reply:
So if we put min level Info there - we will not be able to log Debug